scripts/checkout_web_vault.sh

```diff
@@ -53,4 +53,7 @@ pushd "${VAULT_FOLDER}"
     git fetch --tags --depth 1 origin "${VAULT_VERSION}"
     # Checkout the branch we want
     git -c advice.detachedHead=false checkout FETCH_HEAD
+    # Make sure we don't use non-OSI code
+    # requires patching in next step
+    rm -rf bitwarden_license
```

Comment:
This doesn't seem useful, that directory is not packed or used, only pulled as part of the repo. Removing this doesn't serve anything useful.